recipes/liquorice/conda-recipe/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: liquorice
+  version: 0.5.3
+
+source:
+  url: https://github.com/epigen/LIQUORICE/archive/refs/tags/v0.5.3.tar.gz
+  sha256: 08301c9084e00394b68b1cf4d9aeadfb5d6772dfa3c36e7bb8cbe5bc535bf31d
+
+build:
+  number: 0
+  entry_points:
+    - LIQUORICE = liquorice.LIQUORICE:main
+    - LIQUORICE_summary = liquorice.LIQUORICE_summary:main
+  noarch: python
+  script: {{PYTHON}} setup.py install
+
+requirements:
+  host:
+    - python >=3.7
+  run:
+    - python ==3.7.9
+    - biopython ==1.79
+    - deeptools ==3.5.1
+    - lmfit ==1.0.2
+    - matplotlib-base ==3.1.1
+    - modin  ==0.8.2
+    - numpy ==1.21.2
+    - pandas ==1.1.4
+    - joblib ==1.0.1
+    - pybedtools ==0.8.2
+    - pybigwig ==0.3.18
+    - scikit-learn ==0.24.2
+    - scipy ==1.7.1
+    - swifter ==1.0.9
+    - parallel
+    - seaborn ==0.11.2
+    - pysam ==0.16.0.1
+  run_constrained:
+    - ray-core ==1.1.0
```

Comment:
These can be moved to the `run` section

Reply:
When I had ray-core under "run", the installation failed on macOS, because ray-core is not available for macOS via conda (see https://anaconda.org/conda-forge/ray-core/files). I am therefore keeping it optional (my tool can also run without ray-core, but will be slower). I do need version 1.1.0 of ray-core if installed at all, though, because the program crashes otherwise.  If you have any better ideas I'd be happy to hear!

Reply:
OK, fair enough